### DISCUSSION TITLE
Rate optimization Power BI report

### DISCUSSION THREAD

**Discussion by prabha11-sys** [microsoft/finops-toolkit#839] [Q&A] [answered]
Hello @flanakin,
Can the Rate optimization Power BI report be used without the FinOps hubs. I haven't implemented FinOps hubs, however I would like to identify what savings do we realize through commitments. I understand that these insights are made available through the Rate optimization report. Is there any other way to get these insights(savingsplan savings, RI savings) without using the FinOps hub ?

**Comment by flanakin** [comment, marked as answer]
You have 4 options:
1. As you know, the [Rate optimization report](https://aka.ms/ftk/pbi/RateOptimization) is designed to work with FinOps hubs today. This is the only out of the box option today.
2. The [Cost Management connector report](https://microsoft.github.io/finops-toolkit/power-bi/connector) works with the built-in connector. This has some size limitations, but it has all the same functionality as the Rate optimization report today.
3. If you can hold out for a few weeks, we are hoping to add raw exports support, which would mean you could point the reports to a pure Cost Management export rather than a hub instance. This will be a little slower to load but will ultimately give you what I think you're asking for. See #837 
4. If you can't wait, you can manually update the `ftk_Hub` function in any report to support exported CSV files. This will require a bit of knowledge of Power Query M language but it's totally doable.

**Reply by flanakin** [reply]
FYI - I'm making good headway on #837

**Comment by prabha11-sys** [comment]
Thanks Michael,
I was a bit hesitant to use the cost management connector as I saw the
below warning in the documentation. Is this still valid and will this cause
the numbers to be skewed ?

*The Cost Management connector uses an older API that does not include all
details about savings plans. You will see unused savings plan charges that
will not have identifiable usage for due to this gap. This will skew
numbers, if you have savings plans. Consider using FinOps hubs
<https://microsoft.github.io/finops-toolkit/hubs> to use savings plans.*

**Reply by prabha11-sys** [reply]
Hi Michael, I used the cost management connector report and I cant get the commitment savings for Savings plan. I face the same issue as reported in this thread https://github.com/microsoft/finops-toolkit/issues/604

**Reply by flanakin** [reply]
Correct. Savings plan data isn't complete in the connector. The only solution we have today is FinOps hubs. But I'm making good progress on adding support for raw exports. Stay tuned.

**Reply by flanakin** [reply]
FYI - We added support for raw exports in 0.5.
